Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -196,20 +195,6 @@ class TelemetryWrapper: TelemetryWrapperProtocol, FeatureFlaggable {
         )
     }
 
-    // Sets hashed fxa sync device id for glean deletion ping
-    func setSyncDeviceId() {
-        // Grab our token so we can use the hashed_fxa_uid and clientGUID for deletion-request ping
-        guard let accountManager = RustFirefoxAccounts.shared.accountManager.peek(),
```

Comment:
Turns out this is actually never initialized before this gets called so this code has actually never really worked. Getting rid it of it and applying a solution for both iOS and Android is probably the preferred thing to do here but curious thoughts on it!